File: cash.py
from cs50 import get_float

def main():

    coins = 0
    c = [25, 10, 5, 1]

    while True:
        dollars = get_float("How many change needed in dollars: ")
        if dollars > 0:
            break

    cents = round(dollars*100)
    print(cents)

    for i in c:
        while cents >= i:
            cents = cents - i
            coins += 1

    print (coins)


if __name__ == "__main__":
    main()

File: cash.c
# include <stdio.h>
# include <math.h>
# include <cs50.h>

int main(void)
{
    // Introduce variables
    int coins = 0;
    float dollars;


    // Get user input for number of dollars change needed
    do
    {
        dollars = get_float("Change needed in dollars: ");
    }
    while (dollars < 0);

    // Change dollars into cents
    int cents = round(dollars * 100);

    // Count quarters needed
    while(cents >= 25){
        cents = cents-25;
        coins++;
    }

    // Count dimes needed
    while(cents >= 10){
        cents = cents-10;
        coins++;
    }

    // Count nickles needed
    while(cents >= 5){
        cents = cents-5;
        coins++;
    }

    // Count pennys needed
    while(cents >= 1){
        cents = cents-1;
        coins++;
    }

    // Print total number of coins used
    printf("%i\n", coins);
}

